src/views/record-browser/query-builder/query-builder.html

```diff
@@ -67,13 +67,13 @@
 
 	<label>Filters</label>
 
-	<button
+	<div
```

Comment:
This fix is unrelated to the issue that this feature request (#19) addresses. In the future, create a new issue and a separate pull-request.